scripts/generate-user-email-list.py

```diff
@@ -1,27 +1,61 @@
 """
 
 Usage:
-    scripts/generate-user-email-list.py <data_api_url> <data_api_token>
+    scripts/generate-user-email-list.py <data_api_url> <data_api_token> [--framework=<slug>]
 """
 import csv
 import sys
+import multiprocessing
+from multiprocessing.pool import ThreadPool
 
 from docopt import docopt
 from dmutils.apiclient import DataAPIClient
+from dmutils.audit import AuditTypes
 
 
-def generate_user_email_list(data_api_url, data_api_token):
-    client = DataAPIClient(data_api_url, data_api_token)
+def is_registered_with_framework(client, framework_id):
+    def is_of_framework(audit):
+        return audit['data'].get('frameworkSlug') == framework_id
```

Comment:
Oh, good catch, I think we're missing this check in the stats. But it shouldn't affect numbers right now.